soc/xtensa/intel_adsp/Kconfig

```diff
@@ -63,4 +77,13 @@ config ADSP_TRACE_SIMCALL
 	  of an enclosing simulator process.  All window contents will
 	  remain identical.
 
+config ADSP_XT_GDB
+	bool "Use register definititions for xt-gdb"
+	depends on GDBSTUB
+	default y
+	help
+	  When true, the gdb stub will use register definitions that are exepcted by
+	  xt-gdb. Otherwise, it will use register definitions for the Zephyr
+	  toolchain gdb client.
```

Comment:
Pedantry: would be clearer to name these as "the Cadence xt-gdb debugger" and "the Zephyr SDK gdb".  Not everyone knows that xt-gdb is the proprietary toolchain debugger.